spec/index.html

```diff
@@ -248,5 +248,36 @@ <h2>Basic Concepts</h2>
 
     <p>FIXME.</p>
   </section>
- </body>
+
+  <section id="sec" class="informative">
+    <h2>Security Considerations</h2>
+
+    <p>See the YAML media type registration.</p>
+  </section>
+
+  <section id="int" class="informative">
+    <h2>Interoperability Considerations</h2>
+
+    <p>
+      For general interoperability considerations on the serialization of
+      JSON documents in YAML, see [[YAML]].
+    </p>
+        
+  </section>
+
+  <section id="int" class="informative">
+    <h2>Fragment identifiers</h2>
+
+    <p>
+      YAML media type support both alias nodes and JSON Pointers [[RFC6905]]
```

Comment:
Note: here we could 

- import application/yaml fragid considerations
- replace the text below after investigating yaml-ld specificities.

Reply:
Again, JSON-LD puts this in IANA Considerations and describes it as being equivalent to other RDF syntaxes. The closest analogy is probably with XML and RDFa, where there is overlap between markup identifiers and fragment identifiers.

Note JSON-LD does not have any direct relationship to JSON Pointers.

We do need to consider and reconcile the overlap between the different YAML fragment identifiers, and the use of fragments in RDF, in general, which do not necessarily require that a fragment be directly associated with a marker in the document.

Reply:
@gkellogg my proposal is

1- let's write down the stuff in these sections as specified in https://datatracker.ietf.org/doc/html/rfc8126#section-1.1
2- if the text just results in a series of references to external documents, we can just reference things into the IANA considerations
3- if the text is longer, we reference the document section

WDYT?